scripts/mkfiles.sh

```diff
@@ -688,3 +689,7 @@ elif [ "$1" = "shelley" ]; then
 else
   echo "Default yaml configuration applied."
 fi
+
+chmod -R +rw $ROOT
+#find $ROOT -type d -exec chmod 0777 {} \;
+find $ROOT -type f -name "*.*key" -exec chmod 0400 {} \;
```

Comment:
Why was this need?

Reply:
because when we create this folder some of the files/directories are copied from the nix store and have only read permissions.